libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/extensions/Mappers.kt

```diff
@@ -0,0 +1,31 @@
+@file:JvmName("Mappers")
```

Comment:
We should also add `@file:JvmName("Mappers")` into `Mappers.kt` (`libdirections-offboard`) so Java clients don't need to type `Kt` 😅